HoneyBadgers.ConsoleApp/ApplicationStart.cs

```diff
@@ -68,61 +73,176 @@ private void QuitApp()
-            WriteLine("Finally, you are able to check if you have already watched the movie!");
-            WriteLine("Have fun!:)\n");
-            WriteLine("Credits goes to: Beata Giełbaga, Patryk Wiśniewski, Sebastian Leszczyński and Maciej Tomala");
-            WriteLine("\nPress any key to return to the main menu.");
+            Console.Clear();
+            Console.WriteLine("This application have been created by Honey-Badgers team.");
+            Console.WriteLine("It simply allows you to search for movies by their name, rating or even by the most frequent assignments to users.");
+            Console.WriteLine("You can also put a movie in the list of movies you want to see in the future.");
+            Console.WriteLine("Finally, you are able to check if you have already watched the movie!");
+            Console.WriteLine("Have fun!:)\n");
+            Console.WriteLine("Credits goes to: Beata Giełbaga, Patryk Wiśniewski, Sebastian Leszczyński and Maciej Tomala");
+            Console.WriteLine("\nPress any key to return to the main menu.");
+
+            Console.ReadKey(true);
+        }
+
+        private void RunSearchMenu()
+        {
+            Console.Clear();
+            string prompt = _logo + "\nHow you want to search?";
+            string[] options = { "Search by title",
+                                "Search by rating lower than",
+                                "Search by rating higher than",
+                                "Search by rating between lower and higher",
+                                "Back to Main menu"};
+            Menu menu = new Menu(prompt, options);
+            var selectedItem = menu.Run();
+            switch (selectedItem)
+            {
+                case 0:
+                    RunSearchByTitle();
+                    break;
+                case 1:
+                    RunSearchByRatingLowerThan();
+                    break;
+                case 2:
+                    RunSearchByRatingHigherThan();
+                    break;
+                case 3:
+                    RunSearchByRatingBetweenLowerAndHigher();
+                    break;
+                case 4:
+
+                    break;
+            }
             
-            ReadKey(true);
-            RunMainMenu();
+        }
+        private void RunAddUserMenu()
+        {
+            Console.Clear();
+            var userService = new UserRepository();
+            userService.AddUser();
+
+            Console.WriteLine("\nPress any key to return to the main menu.");
+            Console.ReadKey(true);
+        }
+        private void RunAddMovieMenu()
+        {
+            Console.Clear();
+            var movieRepository = new MovieRepository();
+            //movieService.AddMovie();TODO: POPRAWIC
+            Console.WriteLine("\nPress any key to return to the main menu.");
+            Console.ReadKey(true);
         }
 
-        private void SearchByTheName()
+        private void RunSearchByTitle()
```

Comment:
dla mnie ten wszystkie metody dla search powinny być przeniesione do jakiegoś searchRepository, za dużo tu logiki nie związanej z menu

Reply:
Wyszystkie RunSearch... odpowiadają za obsługę konsoli. Nie mam pomysłu jak to ugryźć by właśnie zrobić jakieś repository ;) na projekcie pomyslimy :) 
Sama logika wyszukania jest w clasie Searcher (trzyba pomyśleć jakąś ładną nazwe)